plugins/Marketplace/stylesheets/plugin-details.less

```diff
@@ -102,5 +97,249 @@
+        }
+
+        h6 {
+            font-size: 12px;
+            line-height: 16px;
+        }
+
+        iframe{
+            width:100%;
+            border: 0;
+            background-image: url('plugins/Morpheus/images/loading-blue.gif');
+            background-repeat: no-repeat;
+            background-position: center;
+        }
+
+        blockquote {
+            margin: 20px 0;
+            padding-left: 1.5rem;
+            border-left: 5px solid #ee6e73;
+        }
+
+        ul, ol {
+            list-style: initial;
+            padding-left: 20px;
+        }
+
+        p, ul, li {
+            text-align: left;
+            line-height: 20px;
+            list-style: initial;
+        }
+
+        p {
+            margin: 0 0 10px;
+        }
+
+        a {
+            color: @theme-color-link;
+        }
+
+        img {
+            max-width: 95%;
+        }
+
+        // styles for main content from marketplace
+        .main-div-readme {
+            display: flex;
+        }
+
+        .left-div-readme {
+            width: 50%;
+        }
+
+        .right-div-readme {
+            flex-grow: 1; width: 50%;
+        }
+
+        .plugin-metadata-part2 {
+            dl {
+                display: flex;
+                flex-direction: column;
+                flex-wrap: nowrap;
+                font-size: 14px;
+                line-height: 22px;
+            }
+
+            dt {
+                font-weight: 700;
+                min-width: 15%;
+                padding-right: 10px;
+            }
+
+            dd {
+                text-align: right;
+            }
+
+            .pair {
+                display: flex;
+                flex-direction: row;
+                justify-content: space-between;
+                padding: 8px 0;
+                border-bottom: 1px solid #aaa;
+
+                &:last-of-type {
+                    border-bottom: 0;
+                }
+            }
+        }
+
+        .plugin-screenshots {
+            .thumbnails {
+                text-align: center;
+
+                figure {
+                    margin: 24px auto;
+                    border: 1px solid #e0e0e0;
+                    display: inline-block;;
+                }
+
+                img {
+                    max-width: 100%;
+                }
+
+                figcaption {
+                    padding: 8px;
+                    background: #e0e0e0;
+                    text-align: left;
+                }
+            }
+        }
+
+        hr {
+            height: 1px;
+            overflow: hidden;
+            background-image: url('plugins/Marketplace/images/divider.svg');
+            border: 0;
+            margin: 36px 0;
+        }
+    }
+
+    .modal-content__footer {
+        padding: 24px;
+        height: 90px;
+        border-top: 1px solid #aaa;
+        display: flex;
+        justify-content: space-between;
+
+        .cta-container {
+            width: 30%;
```

Comment:
Can we make this `width: auto`?

Narrowing the viewport would currently remove parts of the button text, for example displaying "Start Free". If we are only displaying the matomo logo next to it we should in most languages be able to display the complete button, perhaps with a `max-width: 50%` added so it definitely does not flow into the logo?

Reply:
I can see if we can do minimum and maximum width here, the reason I put 30 is for consistency with the card button and not to have the button too wide.
Perhaps one or two more breakpoint related widths would help better here. I'll see what can be done here.